src/Sylius/Bundle/ApiBundle/Serializer/CommandOperationDenormalizer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Serializer;
+
+use Symfony\Component\Serializer\Exception\MissingConstructorArgumentsException;
+use Symfony\Component\Serializer\Normalizer\ContextAwareDenormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+final class CommandOperationDenormalizer implements ContextAwareDenormalizerInterface
+{
+    /** @var DenormalizerInterface */
+    private $objectNormalizer;
+
+    public function __construct(DenormalizerInterface $objectNormalizer)
+    {
+        $this->objectNormalizer = $objectNormalizer;
+    }
+
+    public function supportsDenormalization($data, $type, $format = null, array $context = []): bool
+    {
+        return isset($context['input']['class']);
```

Comment:
I guess that you should check here if denormalization has already been done, similar to normalizer

Reply:
In fact, it's not required, as, as far as I see, denormalizer is always called only once. We also do not have checked in `AddressDenormalizer`